src/draw/sw/lv_draw_sw.c

```diff
@@ -226,6 +226,15 @@ void lv_draw_sw_rgb565_swap(void * buf, uint32_t buf_size_px)
 
 }
 
+void lv_draw_sw_i1_invert(void * buf, uint32_t buf_size) {
+    if(buf == NULL) return;
+
+    uint8_t *byte_buf = (uint8_t *)buf;
+    for(uint32_t i = 0; i < buf_size; i++) {
```

Comment:
Inverting uint32_t could be faster, but I guess it's okay for small monochrome screens